orangegrove.rb

```diff
@@ -0,0 +1,85 @@
+# ORANGE TREE ------------------------------------------------------------------------------->
+class OrangeTree
+
```

Comment:
Hi @khjrtbrg! May I ask why you're making this suggestion? I'm not sure I understand the difference.